object-manipulation-3/main.js

```diff
@@ -0,0 +1,151 @@
+console.log('Lodash is loaded:', typeof _ !== 'undefined');
+var players = [
+  {
+    name: 'Edwin',
+    hand: [],
+    score: 0
+  },
+  {
+    name: 'Steve',
+    hand: [],
+    score: 0
+  },
+  {
+    name: 'Ian',
+    hand: [],
+    score: 0
+  },
+  {
+    name: 'Raymond',
+    hand: [],
+    score: 0
+  }
+
+];
+
+function createCards() {
+  var cardDeck = [];
+  for (var cardNumber = 1; cardNumber < 14; cardNumber++) {
+    for (var suit = 1; suit < 5; suit++) {
```

Comment:
You can vastly simplify this by making an array of the possible cards and an array of the possible suits, and just loop over those arrays.